### DISCUSSION TITLE
@types/office-js - Latest version (v1.0.184) is missing type definition for onActivated in ShapeCollection

### DISCUSSION THREAD

**Discussion by natejcho** [DefinitelyTyped/DefinitelyTyped#53882] [Issues with a @types package]
the index.d.ts file says that the Api set is for ExcelApi 1.9, this api set should include [new methods for ShapeCollection type](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-9-requirement-set). It is clearly missing in the type definitions, please see line 38436 in @types/office-js/index.d.ts

**Comment by ElizabethSamuel-MSFT** [comment]
@natejcho Thanks for letting us know about this.

@alison-mk Can you take a look?

Thanks.

**Comment by alison-mk** [comment]
Hi @natejcho, thanks for submitting an issue. 

I don't see any missing type definitions. In the current [@types/office-js/index.d.ts file](https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js/index.d.ts), I see type definitions for the ExcelApi 1.9 ShapeCollection object starting on line 38436. The definition for `addGeometricShape` is on line 38442, the definition for `addGroup` is on line 38458, and so on. 

Is there a specific ShapeCollection type definition missing from @types/office-js/index.d.ts?  

Thank you, 
Alison
